app/uk/gov/hmrc/cataloguefrontend/search/IndexBuilder.scala

```diff
@@ -49,7 +49,7 @@ class IndexBuilder @Inject()(teamsAndRepositoriesConnector: TeamsAndRepositories
     SearchTerm("explorer", "leaks",             leakRoutes.LeakDetectionController.ruleSummaries.url,                        1.0f, Set("lds")),
     SearchTerm("page",     "whatsrunningwhere", wrwRoutes.WhatsRunningWhereController.releases().url,                        1.0f, Set("wrw")),
     SearchTerm("page",     "deployment",        wrwRoutes.DeploymentHistoryController.history(Environment.Production).url,   1.0f),
-    SearchTerm("page",     "shutter-frontend",  shutterRoutes.ShutterOverviewController.allStates(ShutterType.Frontend).url, 1.0f),
+    SearchTerm("page",     "shutter-overview",  shutterRoutes.ShutterOverviewController.allStates(ShutterType.Frontend).url, 1.0f),
```

Comment:
I think `shutter-frontend` is a better name since the others are `shutter-api` and `shutter-rate`. Probably should at least add `frontend` to the hints.